### PR TITLE
Remove Goerli from supported networks

### LINKED ISSUE #352: Remove Goerli from the supported networks
Goerli is [deprecated](https://ethereum.org/en/developers/docs/networks/#goerli) and will be replaced by [Holesovice](https://github.com/eth-clients/holesovice).

Goerli needs to be removed from the following packages ([semaphore repository](https://github.com/semaphore-protocol/semaphore)):

- [`@semaphore-protocol/data`](https://github.com/semaphore-protocol/semaphore/blob/main/packages/data)
- [`@semaphore-protocol/cli`](https://github.com/semaphore-protocol/semaphore/blob/main/packages/cli)

And from the following repositories:

- https://github.com/semaphore-protocol/subgraph ([`networks.json`](https://github.com/semaphore-protocol/subgraph/blob/main/networks.json), [`README.md`](https://github.com/semaphore-protocol/subgraph/blob/main/README.md))
- https://github.com/semaphore-protocol/website ([`en - deployed-contracts.md`](https://github.com/semaphore-protocol/website/blob/main/versioned_docs/version-V3/deployed-contracts.md), [`es - deployed-contracts.md`](https://github.com/semaphore-protocol/website/blob/main/i18n/es/docusaurus-plugin-content-docs/version-V3/deployed-contracts.md))

However, it makes sense to start working on this issue before releasing Semaphore v4, and in the meantime inform people that Goerli will not be supported anymore soon.

For now a solution can be to add an asterisk to `Goerli`, `Optimism Goerli` and `Arbitrum Goerli` on the [website](https://semaphore.pse.dev/docs/deployed-contracts), add a note like `The Goerli testnet is deprecated` and link the following websites for more information:

* Goerli: https://ethereum.org/en/developers/docs/networks/#goerli
* Optimism Goerli: https://community.optimism.io/docs/useful-tools/networks/#op-goerli 
* Arbitrum Goerli: https://docs.arbitrum.io/for-devs/concepts/public-chains#arbitrum-goerli

### PR DESCRIPTION
**Pull request by cedoor**
## Related Issue

Closes #352

## Does this introduce a breaking change?

-   [x] Yes
-   [ ] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +20,10 @@
         "prettier:write": "prettier -w .",
         "docs": "typedoc --cname js.semaphore.pse.dev --githubPages true",
         "version:bump": "yarn workspaces foreach --no-private version -d ${0} && yarn version apply --all && git commit -am \"chore: v${0}\" && git tag v${0}",
-        "version:publish": "yarn build:libraries && yarn remove:template-files && yarn workspaces foreach --no-private npm publish --tolerate-republish --access public",
+        "version:publish": "yarn build:libraries && yarn clean:cli-templates && yarn workspaces foreach --no-private npm publish --tolerate-republish --access public",
```
Comment (0xjei) [inline package.json:23]: very neat

File: packages/cli/README.md [hunk 2]
```diff
@@ -76,9 +76,9 @@ Options:
 
 Commands:
   create [options] [project-directory]  Create a Semaphore project with a supported template.
-  get-groups [options]                  Get the list of groups from a supported network (e.g. goerli or arbitrum).
-  get-group [options] [group-id]        Get the data of a group from a supported network (e.g. goerli or arbitrum).
-  get-members [options] [group-id]      Get the members of a group from a supported network (e.g. goerli or arbitrum).
-  get-proofs [options] [group-id]       Get the proofs of a group from a supported network (e.g. goerli or arbitrum).
+  get-groups [options]                  Get the list of groups from a supported network (e.g. sepolia or arbitrum).
```
Comment (0xjei) [inline packages/cli/README.md:79]: Shouldn't this be like

```suggestion
  get-groups [options]                  Get the list of groups from a supported network (e.g. sepolia or arbitrum-sepolia).
```

File: packages/cli/src/index.ts [hunk 3]
```diff
@@ -117,7 +117,7 @@ program
 
 program
     .command("get-groups")
-    .description("Get the list of groups from a supported network (e.g. goerli or arbitrum).")
+    .description("Get the list of groups from a supported network (e.g. sepolia or arbitrum).")
```
Comment (0xjei) [inline packages/cli/src/index.ts:120]: Same as the suggestion above

**Review by 0xjei — COMMENTED** [review]
Everything looks good so far! I only noticed one small (potential) oversight in the docs. We just need to wait for the `ethers` fix for  merge :blush:

File: packages/cli/README.md (hunk 2 above)
Comment (cedoor) [reply packages/cli/README.md:79]: I mean arbitrum one (the mainnet) here!

Comment (0xjei) [reply packages/cli/README.md:79]: Ahhhh, got it. Are we supporting `arbitrum-one` too?

Comment (cedoor) [reply packages/cli/README.md:79]: Yes!

Comment (0xjei) [reply packages/cli/README.md:79]: Because I don't see it in the `networks.json` file.

Comment (cedoor) [reply packages/cli/README.md:79]: It's here: https://github.com/semaphore-protocol/semaphore/blob/feat/semaphore-v4/packages/contracts/deployed-contracts/arbitrum.json

**Review by vplasencia — APPROVED** [review]

**Pull request merged**